packages/picasso/src/Select/utils/get-flat-options/get-flat-options.ts

```diff
@@ -0,0 +1,13 @@
+import { Option, OptionGroups } from '../../types'
+import { isOptionsType } from '../../utils'
+
+const getFlatOptions = (options: Option[] | OptionGroups): Option[] => {
```

Comment:
We're not getting but transforming the options here. What do you think about such a name?
```suggestion
const flattenOptions = (options: Option[] | OptionGroups): Option[] => {
```